src/SDKs/SqlManagement/Sql.Tests/FailoverGroupCrudScenarioTests.cs

```diff
@@ -0,0 +1,182 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Management.Resources;
+using Microsoft.Azure.Management.Resources.Models;
+using Microsoft.Azure.Management.Sql;
+using Microsoft.Azure.Management.Sql.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Sql.Tests
+{
+    public class FailoverGroupCrudScenarioTests
+    {
+        public static string DefaultPrimaryLocation
+        {
+            get
+            {
+                return "North Europe";
+            }
+        }
+
+        public static string DefaultSecondaryLocation
+        {
+            get
+            {
+                return "SouthEast Asia";
+            }
+        }
+
+        private Server CreateServer(SqlManagementClient sqlClient, ResourceGroup resourceGroup, string serverPrefix, string location)
+        {
+            string login = "dummylogin";
+            string password = "Un53cuRE!";
+            string version12 = "12.0";
+            string serverName = SqlManagementTestUtilities.GenerateName(serverPrefix);
+            Dictionary<string, string> tags = new Dictionary<string, string>();
+
+            var v12Server = sqlClient.Servers.CreateOrUpdate(resourceGroup.Name, serverName, new Microsoft.Azure.Management.Sql.Models.Server()
+            {
+                AdministratorLogin = login,
+                AdministratorLoginPassword = password,
+                Version = version12,
+                Tags = tags,
+                Location = location,
+            });
+            SqlManagementTestUtilities.ValidateServer(v12Server, serverName, login, version12, tags, location);
+            return v12Server;
+        }
+
+        [Fact]
+        public void TestCreateDeleteFailoverGroup()
+        {
+            string testPrefix = "sqlcrudtest-";
+            string suiteName = this.GetType().FullName;
+            SqlManagementTestUtilities.RunTestInNewResourceGroup(suiteName, "TestCreateDeleteFailoverGroup", testPrefix, (resClient, sqlClient, resourceGroup) =>
+            {
+                // Create primary and partner servers
+                //
+                Server primaryServer = CreateServer(sqlClient, resourceGroup, testPrefix, DefaultPrimaryLocation);
+                Server partnerServer = CreateServer(sqlClient, resourceGroup, testPrefix, DefaultSecondaryLocation);
+
+                // Create a failover group
+                //
+                string failoverGroupName = SqlManagementTestUtilities.GenerateName(testPrefix);
+                var fgInput = new FailoverGroup()
+                {
+                    ReadOnlyEndpoint = new FailoverGroupReadOnlyEndpoint()
+                    {
+                        FailoverPolicy = "Disabled",
```

Comment:
You should be able to write `ReadOnlyEndpointFailoverPolicy.Disabled`. (How do I know? see https://github.com/Azure/azure-rest-api-specs/blob/master/arm-sql/2015-05-01-preview/swagger/failoverGroups.json#L490 )